Install.md

```diff
@@ -87,23 +69,23 @@ This guide helps you build and install Snort.
     ```sh
     cd snort-2.9.8.3/
     ```
-    
-2. Run the configuration script.
+2. Prepare for automake, then autocreate makefile. 
     ```sh
-    ./configure --enable-sourcefire
+    aclocal
+    autoconf
+    autoheader
+    autoreconf -fvi
+    automake -a
     ```
-
-3. Create Patch of Makefille      
-   ```sh
-   cd ../
-   ./patching-Makefile.sh
-   ```
-
+3. Run the configuration script.
+    ```sh
+    ./configure --enable-sourcefire --enable-static
```

Comment:
Let's keep this as `./configure --enable-sourcefire --enable-static --disable-shared` I ran into issues with --disable-shared left out, DPDK complained about conflicting links.